agrona/src/main/java/org/agrona/concurrent/broadcast/BroadcastReceiver.java

```diff
@@ -200,4 +200,15 @@ private boolean validate(final long cursor, final AtomicBuffer buffer, final int
     {
         return (cursor + capacity) > buffer.getLongVolatile(tailIntentCounterIndex);
     }
+
+    public long cursor()
+    {
+        return cursor;
+    }
+
+    public long tailIntentCounterIndex()
+    {
+        VarHandle.acquireFence();
```

Comment:
What is the purpose of the acquire fence? The getLongVolatile has sequential consistent semantics and will provide acquire semantics (when used properly).

Reply:
The tailIntentCounterIndex contains the index of the tailIntentCounter in the buffer. So the method should be named 'tailIntentCounter'.